compiler/onecc-docker/onecc-docker

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env bash
+''''exec "python3" "onecc-docker" "$@" #'''
+
+
+
+import sys
+import subprocess
+import json
+import os
+import re
+import requests
+
+
+def _is_image(version) : 
+    cmd = ['docker', 'images', f'onecc:{version}']
+    p = subprocess.Popen(cmd,  stdout=subprocess.PIPE, stderr=subprocess.STDOUT) 
+    out, err = p.communicate()
+    dic = {}
+
+    decoded_out = [i.split() for i in out.decode().split("\n") if len(i)]
+
+    if len(decoded_out) <2 :
+        return False
+
+    dic[decoded_out[0][0]] = decoded_out[1][0]
+    dic[decoded_out[0][1]] = decoded_out[1][1]
+
+    tag_name = cmd[2].split(':')
+
+    if dic['REPOSITORY'] == tag_name[0] and dic['TAG'] == tag_name[1]  :
+        return True 
+    else :
+        return False
+
+def write_dockerfile(script_path,version) :
+
+    with open(f'{script_path}/Dockerfile', 'w', encoding='utf-8') as dockerfile: 
+        dockerfile.write('FROM ubuntu:18.04 \n\n')
+        dockerfile.write('RUN apt update -y && \ \n')
+        dockerfile.write('\t apt install -y \ \n')
+        dockerfile.write('\t wget && \ \n')
+        dockerfile.write('\t apt clean \n')
+        dockerfile.write(f'RUN wget https://github.com/Samsung/ONE/releases/download/{version}/one-compiler_{version}_amd64.deb && \ \n')
+        dockerfile.write(f'\t apt-get install -y ./one-compiler_{version}_amd64.deb \n')
+        dockerfile.write('ENTRYPOINT ["onecc"]')
+
+
+def main():
+    onecc_version = subprocess.check_output("command -v onecc > /dev/null && echo $(onecc -v)", shell=True ,encoding='utf-8')
+    installed_onecc_version = onecc_version.split(' ')[2]
+
+    script_path = re.sub("\n", "", os.popen('pwd').read())
+    response = requests.get("https://api.github.com/repos/Samsung/ONE/tags")
+    if  response.status_code != 200 :
+        print(response.status_code)
+        print(response.reason)
+        print("we can't get recent tag")
+        exit(1)
+    else :
+        versions_str = response.content
+        versions_json = json.loads(versions_str)
+        recent_version = versions_json[0]["name"]
+
+    if(recent_version == installed_onecc_version):
```

Comment:
This if condition works in the python script?
```suggestion
    if recent_version == installed_onecc_version:
```

Reply:
I missed to erase parparentheses on my if statement. 
It works on python script when I tested, but I'll erase parparentheses

Reply:
Ah, you're right it works. FYI, I think parentheses should be used only if it improves readability or you actually want to change the order of expression calculation.